diff --git a/node_modules/ansi-to-html/lib/ansi_to_html.js b/node_modules/ansi-to-html/lib/ansi_to_html.js
index e8b2db4..c8b4fd2 100644
--- a/node_modules/ansi-to-html/lib/ansi_to_html.js
+++ b/node_modules/ansi-to-html/lib/ansi_to_html.js
@@ -308,7 +308,7 @@ function pushText(text, options) {
   if (options.escapeXML) {
     return entities.encodeXML(text);
   }
-
+  text = text.replace(/ /g, "&nbsp;");
   return text;
 }
 /**
